src/js/components/Calendar/Calendar.js

```diff
@@ -194,32 +194,30 @@ const Calendar = forwardRef(
       // greater than a year. If that's the case, calendar should update without
       // animation.
       let diffBoundsAboveYearFlag = false;
-      let sameDisplayBounds = false;
-      if (nextDisplayBounds[0].getTime() < displayBounds[0].getTime()) {
-        if (
-          displayBounds[0].getTime() - nextDisplayBounds[0].getTime() >
-          millisecondsPerYear
-        ) {
-          diffBoundsAboveYearFlag = true;
-        }
-      } else if (nextDisplayBounds[1].getTime() > displayBounds[1].getTime()) {
-        if (
-          nextDisplayBounds[1].getTime() - displayBounds[1].getTime() >
-          millisecondsPerYear
+      if (nextDisplayBounds !== displayBounds) {
```

Comment:
Aren't these guaranteed to be different, since `buildDisplayBounds()` returns a new array.

Reply:
The thing is: this hook is triggered twice whenever Calendar is showed (and with inline Calendar, it triggers twice in the first render). The idea was to prevent the whole process of updating bounds being executed in this scenario (and any unknown situation this might happen).

Also, thanks for asking that. I did some testing to explain what I did, and discovered that 
`displayBounds === nextDisplayBounds` does not behave as
`nextDisplayBounds[0].getTime() === displayBounds[0].getTime() &&
          nextDisplayBounds[1].getTime() === displayBounds[1].getTime()`

So I'll update that in the refactor. Let me know if you think we should keep this or drop it. :smile: